cypress-io/cypress
# fix: Pass flag to chrome to disable translation prompt when running tests

- Closes https://github.com/cypress-io/cypress/issues/28225

### Additional details

When launching Chrome for testing, we don't want the translation popup to come up. This is one of the documented flags that we reference as ones that may be considerations for automation here: https://github.com/GoogleChrome/chrome-launcher/blob/main/docs/chrome-flags-for-tools.md

### Steps to test

No way to automate this test

I guess you'd have to find a website that triggers translate and see that it pops up with that before and not after. I think this is relatively safe to add as is though.

### How has the user experience changed?

Should no longer see translate popup.

### PR Tasks
